packages/commonwealth/server/controllers/server_communities_methods/update_community.ts

```diff
@@ -210,18 +209,17 @@ export async function __updateCommunity(
       throw new AppError(Errors.InvalidTransactionHash);
     }
 
-    const ownerOfCommunity = addresses.find(
-      (a) => a.community_id === community.id && a.role === 'admin',
-    );
-    if (!ownerOfCommunity) {
+    // XXX: we only permit the community admin and not the site admin to create namespace
+    //   is this correct?
```

Comment:
did we verify this? We allow both community/site admin to update community data in frontend

Reply:
yes this is correct, remove the question part of the comment